python/delta/connect/tests/test_deltatable.py

```diff
@@ -0,0 +1,162 @@
+#
+# Copyright (2024) The Delta Lake Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import unittest
+import sys
+
+from delta.connect.testing.utils import DeltaTestCase
+
+path_to_delta_connect_tests_folder = os.path.dirname(os.path.abspath(__file__))
+path_to_delta_folder = os.path.dirname(os.path.dirname(path_to_delta_connect_tests_folder))
+sys.path.append(path_to_delta_folder)
+
+from tests.test_deltatable import DeltaTableTestsMixin
+
+
+class DeltaTableTests(DeltaTableTestsMixin, DeltaTestCase):
```

Comment:
A little confused, isn't nothing tested here?

Reply:
@allisonport-db https://github.com/delta-io/delta/blob/master/python/delta/tests/test_deltatable.py `test_forPath`, `test_forPathWithOptions, `test_forName`, and `test_alias_and_toDF` but under Delta Connect mode